src/storage.js

```diff
@@ -23,4 +26,41 @@ module.exports = {
   getFeeds: () => redis.smembers(FEEDS),
 
   getFeed: feedID => redis.hgetall(feedID),
+
+  //  addPost
+  addPost: async post => {
+    const postId = await redis.incr(POST_ID);
+    await redis
+      .multi()
+      .hmset(
+        postId,
+        'guid',
+        post.guid,
+        'author',
+        post.author,
+        'title',
+        post.title,
+        'link',
+        post.link,
+        'content',
+        post.content,
+        'text',
+        post.text,
+        'updated',
+        post.updated,
+        'published',
+        post.published,
+        'url',
+        post.url,
+        'site',
+        post.site
+      )
+      .zadd(POSTS, Date.parse(post.published), postId)
+      .exec();
+    return postId;
+  },
+
+  getPosts: () => redis.zrangebyscore(POSTS, 0, +Infinity),
```

Comment:
This is great.  Either in this PR, or in a follow-up, we can support paging here, by allowing the caller to pass arguments to specify which elements in the set to get.  That way they can request a portion vs. everything.